### PR TITLE
Add specs about links and meta information

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

File: api/formatting.md [hunk 1]
```diff
@@ -96,6 +96,95 @@ Example:
...
 ```
 
+### Meta information should be wrapped in meta
+
+Example:
+
+```
+// avoid this
+{
+    "messageId": "115293224292673",
```
Comment (MarcosX) [inline api/formatting.md]: Maybe use a simpler/smaller example?

File: api/formatting.md [hunk 2]
```diff
@@ -96,6 +96,95 @@ Example:
...
+}
+```
+
+### Resource link can be wrapped inside it
+
+Example:
+
+```
+// avoid this
+{
```
Comment (MarcosX) [inline api/formatting.md]: Same here, try to use a smaller/simpler example so it's clear what the specification is about.

File: api/formatting.md (hunk 1 above)
Comment (BrunoQuaresma) [reply api/formatting.md]: Yeah, I will. 

But what do you think about the proposed spec? Does it make sense?

Comment (MarcosX) [reply api/formatting.md]: yep, both are valid and look good to me :)

Comment (BrunoQuaresma) [reply api/formatting.md]: Good. I'm updating the examples.

**Comment by BrunoQuaresma** [comment]
@MarcosX updated!

**Pull request merged**